data/tooltips.txt

```diff
@@ -347,6 +347,12 @@ tip "outfit space needed:"
 tip "outfit space free:"
 	`How much space this ship has for outfits (including weapons and engines).`
 
+tip "overheat damage rate:"
+	`How much hull damage this ship will take per second that it is overheated.`
```

Comment:
Should this mention that the value scales with the amount of excess heat?

Reply:
```suggestion
	`How much hull damage this ship will take per second that it is overheated. Overheat damage scales with how far past the overheat threshold the ship is.`
```